tests/album-delete.test.js

```diff
@@ -0,0 +1,76 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const db = require('../src/db');
+const app = require('../src/app');
+
+describe('Delete Album', () => {
+    let album;
+    let artist;
+    let singleArtistId;
+    beforeEach(async () => {
+        let albumData;
+        let artistData;
+
+        artistData = await Promise.all([
+            db.query(
+                `INSERT INTO Artists (name, genre) VALUES ($1, $2) RETURNING *`, [
+                    'Nevermind',
+                    'rock',
+                ]),
+        ]);
+            artist = artistData.map(({ rows }) => rows[0]);
```

Comment:
You don't need to do a Promise.all here as you are only inserting one entry into the data, also you don't need to map over the results. You could extract the row straight away like you do in artist-delete.test.js